projects/com.oracle.truffle.llvm.test/suites-configs/gccbc/gcc.c-torture/execute/working.exclude

```diff
@@ -0,0 +1,84 @@
+gcc.c-torture/execute/20010518-2.c
```

Comment:
@SwapnilGaikwad told me offline that these files can be processed by the `.ll` parser, but not by the `.bc` parser.